app/models/validations/sales/financial_validations.rb

```diff
@@ -1,14 +1,43 @@
 module Validations::Sales::FinancialValidations
+  include Validations::Sales::SharedValidations
   # Validations methods need to be called 'validate_<page_name>' to run on model save
   # or 'validate_' to run on submit as well
 
   def validate_income1(record)
     if record.ecstat1 && record.income1 && record.ownershipsch == 1
       if record.london_property?
-        record.errors.add :income1, I18n.t("validations.financial.income1.over_hard_max", hard_max: 90_000) if record.income1 > 90_000
+        record.errors.add :income1, I18n.t("validations.financial.income.over_hard_max", hard_max: 90_000) if record.income1 > 90_000
       elsif record.income1 > 80_000
-        record.errors.add :income1, I18n.t("validations.financial.income1.over_hard_max", hard_max: 80_000)
+        record.errors.add :income1, I18n.t("validations.financial.income.over_hard_max", hard_max: 80_000)
       end
     end
+
+    if record.income1 && record.income2
+      if record.london_property?
+        record.errors.add :income1, I18n.t("validations.financial.income.combined_over_hard_max", hard_max: 90_000) if record.income1 + record.income2 > 90_000
+      elsif record.income1 + record.income2 > 80_000
+        record.errors.add :income1, I18n.t("validations.financial.income.combined_over_hard_max", hard_max: 80_000)
+      end
+    end
```

Comment:
I would potentially extract this as something like `validate_comined_income` and add errors to both fields if it falls into the condition, but no strong opinions if you think this is fine

Reply:
Sounds sensible to me

Reply:
I think it would make sense to remove it from here and lines 33-38, but other than the it looks good!

Reply:
Thanks for noticing 🤦‍♂️ I can't believe I forgot to do that